spesifikasjoner/afpant/afpant-intensjon/intensjon-0-1-0.md

```diff
@@ -0,0 +1,82 @@
+# Intensjon om ønsket tinglysingsmetode 
+En bank kan sende forespørsel om tinglysingsmetode til en megler basert på kjøpers fødsels- og personnummer og eiendomsobjektet som skal finansieres. Banken inkluderer sin egen intensjon om tinglysingsmetode i forespørselen. 
```

Comment:
Skal egen (bank) intensjon være optional?
Det er vel egentlig omtalt som "Intensjon fra Bank", vil folk kjenne seg igjen i "Forespørsel om intensjon"?

Reply:
Har jeg gått glipp av en diskusjon her? Trodde man alltid skulle sende sin intensjon når man ba om intensjon. Dvs at bank normalt ikke gjør ett eget kall for intensjon, men ber som intensjon og da samtidig sier hvilken intensjon banken har. Usikker på om vi skal tillate å ikke angi denne. Er ikke det bare å risikere at man angir blank og dermed gjør megler usikker på hva man mener?

Reply:
som Jon Egil sier - tanken var at du alltid oppgir egen intensjon når du spør om mottakers intensjon, derav ikke skille mellom bank/megler i denne sammenheng .

Reply:
Og fra tidligere av har det vært snakket om at man sender sin egen intensjon til motparten - og for mottakers intensjon i retur. I den konteksten har denne meldingen i mitt hode ikke vært sett på som en forespørsel, men en informasjonsmelding. Og så skulle man få mottakers intensjon i retur.